### PR TITLE
The great renaming

### PR DESCRIPTION
**Pull request by kcsongor**
- [x] endpoint -> transceiver
- [x] manager -> nttmanager
- [x] sibling -> peer

### CODE REVIEW

File: evm/src/NttManager.sol [hunk 1]
```diff
@@ -275,30 +275,33 @@ contract Manager is
...
 
-        emit MessageAttestedTo(digest, endpoint, _getEndpointInfosStorage()[endpoint].index);
+        emit MessageAttestedTo(
+            digest, transceiver, _getTransceiverInfosStorage()[transceiver].index
+        );
     }
 
     /*
-     * @dev pause the Endpoint.
+     * @dev pause the Transceiver.
```
Comment (djb15) [inline evm/src/NttManager.sol:291]: Unrelated but this comment is incorrect

**Review by RahulMaganti47 — APPROVED** [review]

File: evm/src/WormholeTransceiver.sol [hunk 2]
```diff
@@ -5,58 +5,58 @@ import "wormhole-solidity-sdk/WormholeRelayerSDK.sol";
...
-contract WormholeEndpoint is Endpoint, IWormholeEndpoint, IWormholeReceiver {
+contract WormholeTransceiver is Transceiver, IWormholeTransceiver, IWormholeReceiver {
     using BytesParsing for bytes;
 
     uint256 public constant GAS_LIMIT = 500000;
     uint8 public immutable consistencyLevel;
 
-    /// @dev Prefix for all EndpointMessage payloads
+    /// @dev Prefix for all TransceiverMessage payloads
     ///      This is 0x99'E''W''H'
```
Comment (djb15) [inline evm/src/WormholeTransceiver.sol:22]: More intrusive, but should we change the payload prefix to match the naming change?

Comment (djb15) [reply evm/src/WormholeTransceiver.sol:22]: Same goes for all these constants actually, the ones below aren't correct anymore (i.e. the comment does not align with the actual bytes)

Comment (kcsongor) [reply evm/src/WormholeTransceiver.sol:22]: good point, let me fix those -- tempted to keep the 'E' 'W' 'H' though. what do you think?

Comment (djb15) [reply evm/src/WormholeTransceiver.sol:22]: Yeah that one is maybe worth keeping as is

File: evm/src/interfaces/INttManagerStandalone.sol
Comment (djb15) [inline evm/src/interfaces/INttManagerStandalone.sol:1]: I'm assuming we don't need this anymore/should rename?

File: solana/programs/example-native-token-transfers/src/peer.rs
Comment (djb15) [inline solana/programs/example-native-token-transfers/src/peer.rs:1]: Any reason why this has moved up a directory? Also the `/endpoints/accounts` folder it came from can be renamed

Comment (kcsongor) [reply solana/programs/example-native-token-transfers/src/peer.rs:1]: I don't think it moved up -- there is still a `transceivers/accounts/peer.rs` (which is what used to be the "endpoint sibling". this thing is the "manager sibling")

Comment (kcsongor) [reply solana/programs/example-native-token-transfers/src/peer.rs:1]: perhaps this file should move into an `accounts` folder

Comment (djb15) [reply solana/programs/example-native-token-transfers/src/peer.rs:1]: Ah yeah my bad I got those mixed up. Up to you whether you think it's worth moving

**Review by djb15 — APPROVED** [review]
My eyes are bleeding now but I think this looks good

**Pull request merged**